lib/xboxrt/libc_extensions/direct.c

```diff
@@ -0,0 +1,85 @@
+// SPDX-License-Identifier: MIT
+
+// SPDX-FileCopyrightText: 2023 ExoSkye
+
+// Part of Microsoft CRT
+
+#include <errno.h>
+#include <string.h>
+#include <stdbool.h>
+#include <windows.h>
+
+// Made referencing https://www.digitalmars.com/rtl/direct.html, retrieved 2023-06-10, copyrighted 1999-2018 by Digital Mars
+
+
+/*
+ * TODO: Make a working directory system?
+ */
+
+int _chdir(char* path) {
```

Comment:
I'd suggest we either remove these stubs or atleast assert within them. We have `assert(0);` is other stubs.

Currently these will silently fail with a generic error code and the end user will have no idea why.

Reply:
Good point, I'll do that